e2e test failing due to update imaging modality names

See: https://github.com/neurobagel/recipes/actions/runs/20380037894/job/58568017528?pr=164

Assertion error:
```bash
  1) When I load the query tool
       I see the expected options for each variable dropdown:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'T1 Weighted' within the element: <ul#:rh:-listbox.MuiAutocomplete-listbox.css-1yq2qy6> but never did.
      at Context.eval (webpack:///./cypress/e2e/fullstack-test.cy.js:62:35)

  2) When I run an unfiltered query on all nodes
       I see the expected matching dataset info:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'fMRI' within the element: <div.MuiPaper-root.MuiPaper-elevation.MuiPaper-rounded.MuiPaper-elevation1.MuiCard-root.css-s18byi> but never did.
      at eval (webpack:///./cypress/e2e/fullstack-test.cy.js:97:23)
  at Array.forEach (<anonymous>)
      at Context.eval (webpack:///./cypress/e2e/fullstack-test.cy.js:96:27)
      at withinFn (http://localhost:3000/__cypress/runner/cypress_runner.js:136019:8)
      at <unknown> (http://localhost:3000/__cypress/runner/cypress_runner.js:136100:16)
      at __WEBPACK_DEFAULT_EXPORT__ (http://localhost:3000/__cypress/runner/cypress_runner.js:133958:10)
      at Context.within (http://localhost:3000/__cypress/runner/cypress_runner.js:136086:67)
      at wrapped (http://localhost:3000/__cypress/runner/cypress_runner.js:141671:19)
```

--- a/cypress/e2e/fullstack-test.cy.js
+++ b/cypress/e2e/fullstack-test.cy.js
@@ -59,7 +59,7 @@ describe('When I load the query tool', () => {
         
         // Imaging modality
         cy.get('[data-cy="Imaging modality-categorical-field"]').click();
-        cy.get('[role="listbox"]').contains("T1 Weighted");
+        cy.get('[role="listbox"]').contains("T1-weighted image");
         cy.get('[data-cy="Imaging modality-categorical-field"]').click();
 
         // Pipeline Name
@@ -92,7 +92,7 @@ describe('When I run an unfiltered query on all nodes', () => {
         cy.get('[data-cy="result-container"]').within(() => {
             cy.get('[data-cy^="card-"]').contains("BIDS synthetic").closest('[data-cy^="card-"]').as("bidsSyntheticCard");
             cy.get("@bidsSyntheticCard").within(() => {
-                const substrings = ["5 subjects match", "5 total subjects", "fMRI", "T1"]
+                const substrings = ["5 subjects match", "5 total subjects", "BOLD", "T1W"]
                 substrings.forEach(substring => (
                     cy.contains(substring, {matchCase: false})
                 ))

